### PR TITLE
adding files for card 1, 2, 4 and 5

### PR DESCRIPTION
**Pull request by shahzadsalamat**

### CODE REVIEW

File: src/components/gb-card-4 - Copy.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          <div className='gb-card-four-sub-card-info'>
+            <h2 className='gb-title-medium'>Astronauts could land on Red Planet by 2039</h2>
+            <ul className='posted-info'>
+              <li className='posted-by'>
+              <SourceSVG classes="gb-label-icon gb-icon-black-opacity-30 gb-icon-small"/>
+                <a className='gb-text-black' href='www.globuzzer.com'>SPACE.com</a>
+              </li>
+              <li className='posted-at'>
+                 <ClockSVG classes="gb-label-icon gb-icon-black-opacity-30 gb-icon-small"/>
+                <p>Apr 5, 2015</p>
```
Comment (biancaaaaa) [inline src/components/gb-card-4 - Copy.js]: the date should be also a prop

File: src/components/gb-card-4 - Copy.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+     
+      <div className='component-to-copy gb-card-four-wrapper'>
+      <ul className='gb-card-four-sub-cards'>
+        <li className='gb-card-four-sub-card'>
+          <div className='gb-card-four-sub-card-info'>
+            <h2 className='gb-title-medium'>Astronauts could land on Red Planet by 2039</h2>
+            <ul className='posted-info'>
+              <li className='posted-by'>
+              <SourceSVG classes="gb-label-icon gb-icon-black-opacity-30 gb-icon-small"/>
+                <a className='gb-text-black' href='www.globuzzer.com'>SPACE.com</a>
```
Comment (biancaaaaa) [inline src/components/gb-card-4 - Copy.js]: SPACE.com should be prob, e.g. named "source"

File: src/components/gb-card-4 - Copy.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import '../style/gb-style.css';
+
+export const CardFour = () => {
+  return (
+     
+      <div className='component-to-copy gb-card-four-wrapper'>
+      <ul className='gb-card-four-sub-cards'>
+        <li className='gb-card-four-sub-card'>
+          <div className='gb-card-four-sub-card-info'>
+            <h2 className='gb-title-medium'>Astronauts could land on Red Planet by 2039</h2>
```
Comment (biancaaaaa) [inline src/components/gb-card-4 - Copy.js]: this should be a prop "title"

File: src/components/gb-card-4 - Copy.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import {SourceSVG} from './svgs/SourceSVG';
+
+import '../style/gb-style.css';
+
+export const CardFour = () => {
+  return (
+     
+      <div className='component-to-copy gb-card-four-wrapper'>
+      <ul className='gb-card-four-sub-cards'>
+        <li className='gb-card-four-sub-card'>
```
Comment (biancaaaaa) [inline src/components/gb-card-4 - Copy.js]: You should make an own component CardFoutLi, like we did at CardOne (and put the <li> elements in there)

File: src/components/gb-card-4 - Copy.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```
Comment (biancaaaaa) [inline src/components/gb-card-4 - Copy.js]: please delete this file

File: src/components/gb-card-1.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../style/gb-style.css';
+const CardOneLi = ({ follower, category, background }) => {
```
Comment (siseer11) [inline src/components/gb-card-1.js]: Make it's own component, it should not leave in here, in the same file with the parent component

File: src/components/gb-card-1.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+    <div className="gb-card-one-avatars-wrapper">
+      <div className="gb-title-small">
+        <p> followers</p>
+      </div>
+      <div className="gb-card-one-follower-avatar">
+        <div className="gb-card-one-avatars-container">
+              {
+                follower.map((item)=>(
+					  	<img  src={item.url} alt={item.alt} className="gb-avatar gb-avatar-small"/>
```
Comment (siseer11) [inline src/components/gb-card-1.js]: you have no key, most likely you get an error from it

File: src/components/gb-card-4 - Copy.js (hunk 5 above)
Comment (rambbusha) [reply src/components/gb-card-4 - Copy.js]: Done

Comment (shahzadsalamat) [reply src/components/gb-card-4 - Copy.js]: done!

File: src/components/gb-card-1.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+export const CardOne = ({ card }) => {
+
+  
+  return (
+   <div className="gb-card-one-wrapper">
+      <ul className="gb-card-one">
+       {
+         card.map((cardlist) => (
+        <CardOneLi
```
Comment (siseer11) [inline src/components/gb-card-1.js]: Again you need the key, otherwise you get the errors

File: src/components/gb-card-2.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../style/gb-style.css';
+import PropTypes from 'prop-types';
+
+export const CardTwo = ({ category, background, followerCount, follower }) => {
```
Comment (siseer11) [inline src/components/gb-card-2.js:5]: indentation and blank lines, clean them

File: src/components/gb-card-4.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ClockSVG} from './svgs/ClockSVG';
+import {SourceSVG} from './svgs/SourceSVG';
+
+import '../style/gb-style.css';
+
+const CardfourList = ({ heading, source, date, image }) => {
```
Comment (siseer11) [inline src/components/gb-card-4.js]: Indentation , blank lines , keys on the list , the list should stay in other component, not in the same file

File: src/components/gb-card-1.js (hunk 7 above)
Comment (shahzadsalamat) [reply src/components/gb-card-1.js]: Done!

File: src/components/gb-card-1.js (hunk 8 above)
Comment (shahzadsalamat) [reply src/components/gb-card-1.js]: Done!

File: src/components/gb-card-2.js (hunk 9 above)
Comment (shahzadsalamat) [reply src/components/gb-card-2.js:5]: Done!

File: src/components/gb-card-4.js (hunk 10 above)
Comment (shahzadsalamat) [reply src/components/gb-card-4.js]: Done!

File: src/components/gb-card-1.js (hunk 6 above)
Comment (shahzadsalamat) [reply src/components/gb-card-1.js]: Done!

**Pull request merged**